examples/advanced/camera_random_trajectories/main.py

```diff
@@ -0,0 +1,65 @@
+import blenderproc as bproc
+import numpy as np
+import argparse
+
+from blenderproc.python.utility.SetupUtility import SetupUtility
+
+# Install bop_toolkit_lib
+SetupUtility.setup_pip(["git+https://github.com/thodan/bop_toolkit"])
+from bop_toolkit_lib import transform
+
+parser = argparse.ArgumentParser()
+parser.add_argument('scene', nargs='?', default="examples/resources/scene.obj", help="Path to the scene.obj file")
+parser.add_argument('output_dir', nargs='?', default="examples/advanced/camera_random_trajectories/output", help="Path to where the final files, will be saved")
+args = parser.parse_args()
+
+bproc.init()
+
+# load the objects into the scene
+objs = bproc.loader.load_obj(args.scene)
+
+# define a light and set its location and energy level
+light = bproc.types.Light()
+light.set_type("POINT")
+light.set_location([5, -5, 5])
+light.set_energy(1000)
+
+# Find point of interest, all cam poses should look towards it
+poi = bproc.object.compute_poi(objs)
+
+# Add translational random walk on top of the POI
+poi_drift = bproc.sampler.random_walk(total_length = 25, dims = 3, step_magnitude = 0.005, 
+                                      window_size = 5, interval = [-0.03, 0.03], distribution = 'uniform')
+
+# Rotational camera shaking as a random walk: Sample an axis angle representation
+camera_shaking_rot_angle = bproc.sampler.random_walk(total_length = 25, dims = 1, step_magnitude = np.pi/32, window_size = 5,
+                                                     interval = [-np.pi/6, np.pi/6], distribution = 'uniform', order = 2)
+camera_shaking_rot_axis = bproc.sampler.random_walk(total_length = 25, dims = 3, window_size = 10, distribution = 'normal')
+camera_shaking_rot_axis /= np.linalg.norm(camera_shaking_rot_axis, axis=1, keepdims=True)
+
+for i in range(25):
+    # Camera trajectory that defines a quater circle at constant height 
+    location_cam = np.array([10*np.cos(i/25 * np.pi), 10*np.sin(i/25 * np.pi), 8])
+    # Compute rotation based on vector going from location towards poi + drift
+    rotation_matrix = bproc.camera.rotation_from_forward_vec(poi + poi_drift[i] - location_cam)
+    # random walk axis-angle -> quaternion
+    q_rand = transform.quaternion_about_axis(camera_shaking_rot_angle[i], camera_shaking_rot_axis[i])
+    # quaternion -> rotation matrix
+    R_rand = transform.quaternion_matrix(q_rand)[:3,:3]
```

Comment:
We could use mathutils for this, which would eliminate the dependency to the bop toolkit here
```python
import mathutils
# random walk axis-angle -> rotation matrix
R_rand = np.array(mathutils.Matrix.Rotation(camera_shaking_rot_angle[i], 3, camera_shaking_rot_axis[i]))
```

Reply:
good point